test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2396,16 +2436,29 @@ public void testMetadataColumns_blobNull() throws Exception {
   }
 
   @Test
-  public void testMetadataColumns_array() throws Exception {
-    assumeFalse("MySQL does not support arrays", is(MYSQL));
+  public void testMetadataColumns_array_Oracle() throws Exception {
+    assumeTrue(is(ORACLE));
```

Comment:
I think it might be more readable and true to the spirit of these tests to just use an if in the one test method:

if (is(ORACLE)) { ... } else if (is(H2)) { ... } else { assumeTrue(false); }

I don't know if there's a better way to say assumeTrue(false).